misc/keyvaluestorage/src/node-js/level_db.ts

```diff
@@ -0,0 +1,22 @@
+let database
+
+export default {
+  create(db: string) {
+    if(!database) {
+      if(db == ':memory:') {
```

Comment:
Identity over equality assertion:

```suggestion
      if(db === ':memory:') {
```

Reply:
Also spacing seems to be off, maybe missing prettier run/auto-format here